pkg/pxc/app/statefulset/node.go

```diff
@@ -258,7 +262,8 @@ func (c *Node) Volumes(podSpec *api.PodSpec, cr *api.PerconaXtraDBCluster) (*api
 	if cr.CompareVersionWith("1.3.0") >= 0 {
 		vol.Volumes = append(
 			vol.Volumes,
-			app.GetConfigVolumes("auto-config", "auto-"+ls["app.kubernetes.io/instance"]+"-"+ls["app.kubernetes.io/component"]))
+			app.GetConfigVolumes("auto-config", "auto-"+ls["app.kubernetes.io/instance"]+"-"+ls["app.kubernetes.io/component"]),
+			app.GetSecretVolumes("keyring", "keyring", false))
```

Comment:
should be added only if >= 1.4.0

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/341/commits/3b2a7419889f5c29a1d7e4030aac1ac640d8689e